lib/cocaine/command_line/runners/posix_runner.rb

```diff
@@ -4,10 +4,15 @@ module Cocaine
   class CommandLine
     class PosixRunner
       def self.available?
-        require 'posix/spawn'
-        true
-      rescue LoadError
-        false
+        return @available unless @available.nil?
+
+        @available ||=
```

Comment:
This is a bit of a nitpick, but I really dislike assigning the value from a whole block, but I do like this refactoring. Can you pull the actual `begin`/`end` block out into another (preferably private) method so this assignment is just one line?